legacy/src/scss/_patterns_numa.scss

```diff
@@ -11,7 +11,7 @@
     background-repeat: no-repeat;
     background-size: map-get($icon-sizes, default);
     margin: 0;
-    padding: 0 $sph-inner--x-large 0 0;
+    padding: 0 2.5rem 0 0;
```

Comment:
this should ideally be a calculation of the spaces involved - e.g. icon space plus gaps on left and right - but I'm not sure where it appears so I've just hardcoded it for now. if someone can point me to it, I will replace with the correct calculation.